pkgs/top-level/release.nix

```diff
@@ -235,6 +235,8 @@ let
       zsh = linux;
       zsnes = ["i686-linux"];
 
+      emacs24PackagesNg = packagePlatforms pkgs.emacs24PackagesNg;
```

Comment:
@peti Is this the right thing to do so that Hydra will still build these packages?